packages/blocks/src/store/selectors.js

```diff
@@ -107,6 +107,17 @@ export function getCategories( state ) {
 	return state.categories;
 }
 
+/**
+ * Returns all the available collections.
+ *
+ * @param {Object} state Data state.
+ *
+ * @return {Array} Collections list.
```

Comment:
Based on the state shape, I don't think this returns an array. Should it be returning an array though? That seems like what I might expect to receive as a consumer.